### DISCUSSION TITLE
"Scalable Synthesis of Regular Expressions From Only Positive Examples"

### DISCUSSION THREAD

**Discussion by jariji** [pemistahl/grex#159] [Show and tell]
Just for interest, I saw a new paper doing a similar task

- [paper](https://github.com/limpa105/RegexPlus/blob/main/Regex%2B:%20Synthesizing%20Regular%20Expressions%20from%20Positive%20Examples.pdf)
- [talk](https://www.youtube.com/watch?v=lIfRNjPU12I) 
- repo: https://github.com/limpa105/RegexPlus

**Comment by pemistahl** [comment]
Thank you @jariji, this looks interesting. I will study the paper as soon as I find the time.
